docs/zephyr_02_functional_requirements.sdoc

```diff
@@ -308,12 +308,12 @@ COMPONENT: C Library
 REFS:
 - TYPE: Parent
   VALUE: ZEP-CLIB-001
-TITLE: Selected C libraries documentation in the Zephyr Safety Manual
+TITLE: Document set of Zephyr OS required C librariy functions in Safety Manual
 STATEMENT: >>>
-The set of C Libraries supported by Zephyr needs to be defined documented in the Zephyr Safety Manual
+The set of C Library functions required by Zephyr needs to be documented in the Zephyr Safety Manual
 <<<
 USER_STORY: >>>
-As a Zephyr OS user implementing a safety application I want to have documented which C library functions are supported.
+As a Zephyr user, I want to know which C library functions are being called by the Zephyr OS safety scope code.
```

Comment:
The new statement is a very different scope than the old one.  AFAICT, generating the Safety Artifacts will require ZP to provide a safety-certified implementation of the C library functions used. Documenting the safety-certifiable subset provided is also important for end users.

Reply:
In this case the change of the scope in this req. is intended. The thing here is, that there will be no generated artifact because zephyr will always be a part system which need to be certified in a complete state. To minimize the initial certification effort, the c library functions are not in scope right now. Also we don't know what toolchains / libs will be used in the downstream safety-critical systems, we decided that this needs to be taken care of the integrator who wants to build a safety-critical system with zephyr.